src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2576,7 +2576,9 @@ internal static void WriteConsole(ConsoleHandle consoleHandle, ReadOnlySpan<char
                     {
                         var endOfLine = Environment.NewLine.AsSpan();
                         var endOfLineLength = endOfLine.Length;
+#pragma warning disable CA2014
                         Span<char> outBufferLine = stackalloc char[outBuffer.Length + endOfLineLength];
+#pragma warning restore CA2014
```

Comment:
Shall we open issues tracking this? It looks we shouldn't use `stsackalloc` in a real loop (`do ... while(false)` is fine).

Reply:
See https://github.com/PowerShell/PowerShell/issues/13531